src/server/req_stat.c

```diff
@@ -748,6 +749,45 @@ req_stat_svr(struct batch_request *preq)
 		(void)reply_send(preq);
 }
 
+/**
+ * @brief
+ * 		status_sched - Build the status reply for single scheduler
+ *
+ * @param[in]	psched	-	ptr to sched receiving status query
+ * @param[in]	preq	-	ptr to the decoded request
+ * @param[in,out]	pstathd	-	head of list to append status to
+ *
+ * @return	int
+ * @retval	0	: success
+ * @retval	!0	: PBSE error code
+ */
+static int
+status_sched(pbs_sched *psched, struct batch_request *preq, pbs_list_head *pstathd)
+{
+	int		   rc = 0;
+	struct brp_status *pstat;
+	svrattrl	  *pal;
+
+	pstat = (struct brp_status *)malloc(sizeof(struct brp_status));
+	if (pstat == (struct brp_status *)0)
+		return (PBSE_SYSTEM);
+
+	pstat->brp_objtype = MGR_OBJ_SCHED;
+	(void)strcpy(pstat->brp_objname, psched->sc_name);
```

Comment:
Fixed. Used strncpy.  

Also, I think we need to file a ticket to fix all strcpy.  There are many places overflow was seen.